core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/GraphQlServer.scala

```diff
@@ -19,10 +19,13 @@ object GraphQlServer {
 
   implicit private val SeriesType: ObjectType[Unit, Series] = deriveObjectType[Unit, Series]()
   implicit private val ConsignmentType: ObjectType[Unit, Consignment] = deriveObjectType[Unit, Consignment]()
+  implicit private val FileType: ObjectType[Unit, File] = deriveObjectType[Unit, File]()
 
   private val ConsignmentNameArg = Argument("name", StringType)
   private val ConsignmentIdArg = Argument("id", IntType)
   private val SeriesIdArg = Argument("seriesId", IntType)
+  private val filePathArg = Argument("path", StringType)
```

Comment:
Really minor, but this constant should be called `FilePathArg` for consistency.

(I realise I haven't been very consistent with naming conventions in general, sorry - we should add a linter at some point.)